Realm/Realm.Unity/Runtime/tvOS/Device/realm-wrappers.framework.meta

```diff
@@ -0,0 +1,90 @@
+fileFormatVersion: 2
+guid: 2dac6057a35144954ba99c573dcd9f38
+folderAsset: yes
+PluginImporter:
+  externalObjects: {}
+  serializedVersion: 2
+  iconMap: {}
+  executionOrder: {}
+  defineConstraints: []
+  isPreloaded: 0
+  isOverridable: 1
+  isExplicitlyReferenced: 0
+  validateReferences: 1
+  platformData:
+  - first:
```

Comment:
This file is actually important even though it's kind of difficult to review - it indicates that the tvOS wrappers are disabled on all platforms. This is because they're enabled on build by the logic in UnityWeaver.cs. This is pretty similar to what we're doing with the iOS wrappers.